### PR TITLE
push container through push step

### PR DESCRIPTION
**Pull request by flomotlik**

### CODE REVIEW

**Comment by flomotlik** [comment]
@bfosberry when running this container locally with --push=true and --tag=master I get the following: 
```
Image Push error for image gcr.io/codeship-production/google-deployment-example:latest, Post https://gcr.io/v2/codeship-production/google-deployment-example/blobs/uploads/: token auth attempt for registry: https://gcr.io/v2/token?scope=repository%3Acodeship-production%2Fgoogle-deployment-example%3Apush%2Cpull&service=gcr.io request failed with status: 403 Forbidden
```

The variables are set up correctly though, so it should be working fine from everything I saw

File: codeship-services.yml [hunk 1]
```diff
@@ -10,4 +10,8 @@ deployexample:
   build:
     path: test
     dockerfile_path: Dockerfile.example
-    image: codeship/google-deployment-example
+    image: gcr.io/codeship-production/google-deployment-example
```
Comment (bfosberry) [inline codeship-services.yml]: Try adding 

```
dockercfg_service: gcr_dockercfg
```

to the googleclouddeployment service

**Comment by flomotlik** [comment]
@bfosberry this works now. Could you review?

**Comment by bfosberry** [comment]
Lgtm, but I dont think you need 'add_docker' anymore,can you verify?

**Comment by flomotlik** [comment]
@bfosberry works without add_docker. I'll merge then

**Pull request merged**